tenablesc/report.go

```diff
@@ -0,0 +1,74 @@
+package tenablesc
+
+import (
+	"fmt"
+)
+
+const reportEndpoint = "/report"
+
+// Report represents request/response structure from https://docs.tenable.com/tenablesc/api/Report.htm
+type Report struct {
+	BaseInfo
+	ReportDefinitionID string              `json:"reportDefinitionID"`
+	JobID              string              `json:"jobID"`
+	Type               string              `json:"type"`
+	Status             string              `json:"status"`
+	Running            string              `json:"running"`
+	ErrorDetails       string              `json:"errorDetails"`
+	TotalSteps         string              `json:"totalSteps"`
+	CompletedSteps     string              `json:"completedSteps"`
+	StartTime          UnixEpochStringTime `json:"startTime"`
+	FinishTime         UnixEpochStringTime `json:"finishTime"`
+	OwnerGID           string              `json:"ownerGID"`
+	PubSites           []struct {
```

Comment:
Can this just be `[]BaseInfo` instead of a struct that nests it?

Reply:
addressed in https://github.com/palantir/tenablesc-client/pull/28/commits/04d199ed5fa57bf5cc1acd9344dba94387dadfb4